### ISSUE TITLE
string with tabs got wrong textSize 

### ISSUE THREAD

**Issue by typemytype** [typemytype/drawbot#206]
```python
t = "\thello\tworld"

tabs((70, "left"))
w, h = textSize(t)

text(t, (10, 10))
stroke(0)
fill(None)
rect(10, 10, w, h)
```

see http://www.drawbot.com/content/text/textProperties.html?highlight=tabs#drawBot.tabs

**Comment by justvanrossum** [comment]
Should have been a PR if you leave the tests in a failing state...

**Comment by typemytype** [comment]
fixed in  403a98a857010235e30483aa6bc28637227ce9a4
